css/home.css

```diff
@@ -0,0 +1,96 @@
+@import './library.css';
+@import './header.css';
+
+/* Landing img and text */
+.landing{
+      width: 100%;
+      height: 100vh;  
+      
+}
+.landing::before{
+      background-image: url(../assets/landing-image.jpg);
+      background-repeat: no-repeat;
+      background-size: cover;
+      content: "";
+      z-index: -1;
+      width: 100%;
+      height: 100vh;
+      position: absolute;
+      filter: blur(2px);
+}
+.landing-content{
+      height: 100%;
+      margin: auto;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+      max-width: 70%;
+      gap: var(--spacing-s);
+}
+.landing-btn{
+      color: var(--white-color);
+      font-size: 2.4rem;
+}
+
+/* category section */
+.category-container{
+      display: flex;
+      justify-content: center;
+      flex-wrap: wrap;
+      gap: var(--spacing-s);
+}
+.category-card{
+      min-width: 27rem;
+      min-height: 23rem;
+      width: fit-content;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      background: no-repeat center center;
+      background-size: cover;
+      box-shadow: inset 0 0 200px #000;
```

Comment:
you can use color variable

Reply:
thanks for the suggestion I forgot to use a color variable.